debugview/src/debug/kotlin/com/revenuecat/purchases/debugview/ContextExtensions.kt

```diff
@@ -0,0 +1,14 @@
+package com.revenuecat.purchases.debugview
+
+import android.app.Activity
+import android.content.Context
+import android.content.ContextWrapper
+
+fun Context.findActivity(): Activity {
+    var context = this
+    while (context is ContextWrapper) {
+        if (context is Activity) return context
+        context = context.baseContext
+    }
+    error("no activity")
```

Comment:
Here I'm assuming there will always be an activity... I guess we could handle this and fail gracefully, but the debug view should always be used within a compose project that has an activity, and this is only for debug, so I thought it was ok. Lmk if you think otherwise.